tutorial/01_basic/a_load_examples.py

```diff
@@ -59,3 +59,23 @@
 )
 pl.camera_position = [(194.6, -141.8, 182.0), (34.5, 61.0, 32.5), (-0.229, 0.45, 0.86)]
 pl.show()
+
+###############################################################################
+# Exercise #1 - Use PyVista Examples
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Visualize one of PyVista's built in examples.
+#
+# If your IDE supports it, you should be able to type
+# ``dataset examples.download_`` and press tab to see all the available
+# examples you can download.
+
```

Comment:
I would like to unify how the answers are placed throughout. In my opinion, it would be better to place the answer script in a subfolder and then use the load command to display it. 
```suggestion

# %%load /path/to/answer.py 
```
This is to help those who will view the tutorial later.
@akaszynski  @banesullivan  What do you think?

Reply:
I just spoke about this with @banesullivan, while this is a great idea, if people download the script or notebook, it won't pull down other files, like the `answer.py`.

Instead, let's make a "solved" notebook with the solution. That way it's both rendered online, and they can pull it down on their computer.